src/components/Layout/index.jsx

```diff
@@ -1,10 +1,20 @@
 import React from "react";
 import Header from "../Header";
-// import Footer from '../Footer';
+import { Helmet } from "react-helmet";
 
-export default function Layout({ children }) {
+export default function Layout({ children, title, location }) {
+  console.log(location);
   return (
     <>
+      <Helmet>
+        <meta charSet="utf-8" />
+        <title>{`Bearcabulary ${title && "|" + title}`}</title>
```

Comment:
I think `•` would look better than `|`.
Could you add a space after the divider?

Reply:
![image](https://user-images.githubusercontent.com/75239264/132911452-1519d5bc-34df-4f55-94e7-1d47effda26c.png)
This is what it looks like on the homepage. Needs fix.